src/wp-includes/cache.php

```diff
@@ -102,6 +102,110 @@ function wp_cache_flush() {
 
 	return $wp_object_cache->flush();
 }
+/**
+ * Removes all cache items in a group.
+ *
+ * @since 6.0.0
+ *
+ * @see WP_Object_Cache::flush_group()
+ * @global WP_Object_Cache $wp_object_cache Object cache global instance.
+ *
+ * @param string|array $group name(s) of group to remove from cache.
+ *
+ * @return bool True on success, false on failure group not found.
+ */
+function wp_cache_flush_group( $group, $clear_linked = true ) {
+
+	global $wp_object_cache;
+
+	if ( ! wp_cache_supports_flushing_groups() ) {
+
+		return false;
+	}
+
+	// if group is an array loop and call each key in the array
+	if ( is_array( $group ) ) {
+
+		array_map( 'wp_cache_flush_group', array_values( $group ) );
+
+		return true;
+	}
+
+	// these are linked cache groups, so we have to flush them both if one is called
+	$cache_pairs = wp_cache_get_linked_meta();
+	if ( $clear_linked && isset( $cache_pairs[ $group ] ) ) {
+		foreach ( $cache_pairs[ $group ] as $cache_pair ) {
+			wp_cache_flush_group( $cache_pair, false );
+		}
+	}
```

Comment:
I strongly believe that all this linked group logic should be removed. I don't believe it makes sense. If you want to clear a group, you should just call `wp_cache_flush_group` multiple times. Similar to [this](https://github.com/WordPress/wordpress-develop/blob/ea1fe31f028156a56eb97e8b7cce3d0f0e6e242f/src/wp-includes/post.php#L7400-L7401). 

Flushing this linked groups may have side effects we do not wish for. Let's remove this for now. Maybe we could add it back later.

Reply:
removed